aml-development/ozp-react-commons
# Notification folder share refactor

Make sure to pull latest backend branch

This builds upon the other PR https://github.com/aml-development/ozp-react-commons/pull/91 . The changes introduced by this PR are 

- Updated styling and notification click functionality to make an unread notification appear much different than a read notification
- Refactored notification content into a shared class so that the text for notifications is the same in the drop down and in the modal
- Corrected the issue with the folder share not working

To test: Jira ticket #460:
(need to have center running and updated ozp-react-commons to point to this branch)
- Create two new notifications for your user
- Click on the drop down
- Both should have bold titles and a blue bar to the left of the notification to indicate that they are unread
- Click on one of the notifications
- The notification drop down should not close and the notification should no longer have a blue bar and should have a less bold title

To test: Jira ticket # 658
(need to have hud running and updated ozp-react-commons to point to this branch)
- Log on as bigbrother
- Share a folder with wsmith
-  Log on as wsmith
- See notifications
- Choose the Add button
OLD RESULTS - Nothing
EXPECTED RESULTS - it should add the folder to the recipients HUD screen
